app/models/log_spreadsheet.rb

```diff
@@ -80,12 +80,7 @@ def create_spreadsheet(columns, logs)
       end
       row = sheet.row(row_idx)
       columns.each do |col|
-        if TIME_COLS.include?(col)
-          val = log[col].to_datetime
-        else
-          val = log.value(col)
-        end
-        row.push(val)
+        row.push(log.value(col))
```

Comment:
Out of curiosity, was it causing some issues? I think I might have added similar code to RIGSE.